packages/suite/src/components/suite/NavigationBar/components/MainNavigation/index.tsx

```diff
@@ -68,6 +68,31 @@ const ItemTitle = styled.span<ComponentProps>`
     color: ${props => props.theme.TYPE_LIGHT_GREY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
 
+    &:after {
+        content: '';
+        position: absolute;
+        top: -4px;
+        left: -4px;
+        right: -4px;
+        bottom: -4px;
+        z-index: -1;
+        border-radius: 8px;
+        transition: all 150ms ease-out;
```

Comment:
see the comment above

Reply:
done

Reply:
dtto https://github.com/trezor/trezor-suite/pull/3829#discussion_r655227034